packages/mdc-textfield/foundation.js

```diff
@@ -128,11 +128,11 @@ class MDCTextFieldFoundation extends MDCFoundation {
    * @param {!Event} evt
    */
   handleTextFieldInteraction(evt) {
-    if (this.adapter_.getNativeInput().disabled) {
+    if (this.isDisabled()) {
       return;
     }
-
-    this.receivedUserInput_ = true;
+    const input = this.adapter_.getInputFoundation();
+    input.setReceivedUserInput(true);
```

Comment:
Can this setReceivedUserInput be handled internally by the MDCTextFieldInputFoundation?

Reply:
Changed the input method to handleTextFieldInteraction()